proto/one/common/core/enum_security_provider.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+option csharp_namespace ="ONE.Models.CSharp";
+
+enum EnumSecurityProvider {
+    SECURITY_PROVIDER_UNKNOWN = 0;
+    SECURITY_PROVIDER_LOCAL = 1;
+    SECURITY_PROVIDER_GOOGLE= 2;
+    SECURITY_PROVIDER_AZURE_ACTIVE_DIRECTORY = 3;
+    SECURITY_PROVIDER_KOCH_ID = 4;
```

Comment:
Do we need an entry for our normal IdentityServer here?

Reply:
The local entry is for our normal IdentityServer.
My next story will add a column to the Users table to story the IdentityProviderType.